inc/admin.php

```diff
@@ -81,6 +81,7 @@ public function inline_bootstrap_script() {
 		$default_network       = ( $this->settings->get( 'network_optimization' ) === 'enabled' );
 		$retina_ready          = ! ( $this->settings->get( 'retina_images' ) === 'enabled' );
 		$scale_is_disabled     = ( $this->settings->get( 'scale' ) === 'enabled' );
+		$video_lazy_placeholder = OPTML_URL . 'assets/img/optml_loading.gif';
```

Comment:
here to avoid an extra request, can't we use an SVG loading icon inline? or a base64 encoded image.

Reply:
yes, I'm looking for one to add now